projects/batfish/pom.xml

```diff
@@ -53,6 +53,7 @@
             <configuration>
               <includes>**/CiscoLexer.g4,**/CiscoParser.g4</includes>
               <libDirectory>${basedir}/src/main/antlr4/org/batfish/grammar/cisco</libDirectory>
+              <statusDirectory>${project.build.directory}/maven-status/antlr4-cisco</statusDirectory>
```

Comment:
Yeah something about how I'm installing the plugin is breaking vs working when installing dependencies.